src/schnetpack/interfaces/ase_interface.py

```diff
@@ -341,6 +341,289 @@ def calculate(
+            neighbor_list (schnetpack.transform.Transform): SchNetPack neighbor list
+            energy_key (str): name of energies in model (default="energy")
+            force_key (str): name of forces in model (default="forces")
+            stress_key (str): name of stress tensor in model. Will not be computed if set to None (default=None)
+            energy_unit (str, float): energy units used by model (default="kcal/mol")
+            position_unit (str, float): position units used by model (default="Angstrom")
+            device (torch.device): device used for calculations (default="cpu")
+            input_dtype (torch.dtype): select model input precision (default=float32)
+            output_dtype (torch.dtype): select model output precision (default=float64)
+            converter (callable): converter used to set up input batches
+            transforms (schnetpack.transform.Transform, list): transforms for the converter. More information
+                can be found in the AtomsConverter docstring.
+            uncertainty_fn (callable): Function to compute uncertainty. If not provided, defaults to AbsoluteUncertainty.
+            **kwargs: Additional arguments for basic ase calculator class
+        """
+        # Initialize the parent class without loading a model
+        Calculator.__init__(self, **kwargs)
+
+        self.neighbor_list = deepcopy(neighbor_list)
+        self.device = device
+        self.dtype = dtype
+        self.converter = converter(
+            neighbor_list=neighbor_list,
+            device=device,
+            dtype=dtype,
+            transforms=transforms,
+        )
+
+        self.energy_key = energy_key
+        self.force_key = force_key
+        self.stress_key = stress_key
+
+        # Mapping between ASE names and model outputs
+        self.property_map = {
+            self.energy: energy_key,
+            self.forces: force_key,
+            self.stress: stress_key,
+        }
+
+        # set up basic conversion factors
+        self.energy_conversion = convert_units(energy_unit, "eV")
+        self.position_conversion = convert_units(position_unit, "Angstrom")
+
+        # Unit conversion to default ASE units
+        self.property_units = {
+            self.energy: self.energy_conversion,
+            self.forces: self.energy_conversion / self.position_conversion,
+            self.stress: self.energy_conversion / self.position_conversion**3,
+        }
+
+        # Load multiple models
+        self.models = [self._load_model(model_file) for model_file in models]
+
+        if uncertainty_fn is None:
+            self.uncertainty_fn = self.AbsoluteUncertainty
+        else:
+            self.uncertainty_fn = uncertainty_fn.__get__(self, type(self))
+
+    def calculate(
+        self,
+        atoms: ase.Atoms = None,
+        properties: List[str] = ["energy"],
+        system_changes: List[str] = all_changes,
+        energy_weight: float = 0,
```

Comment:
`xxx_weight` can be removed here, because it is already defined in `__init__` of uncertainty classes